main/design_documents/ARM_NEON_SVE_BRIDGE.rst

```diff
@@ -0,0 +1,78 @@
+=======================================
+Design Document for ARM_NEON_SVE_BRIDGE
+=======================================
+
+This document describes the design of the intrinsics defined in
+("NEON_SVE bridge")<../acle.rst#neon-sve-bridge>. These C intrinsics enable the
```

Comment:
Sorry, the linkin this line seems to be broken (see https://github.com/MDevereau/acle/blob/next-release/main/design_documents/ARM_NEON_SVE_BRIDGE.rst#design-document-for-arm_neon_sve_bridge).

I *think* you might want to replace it with the following one:
```suggestion
`NEON_SVE bridge <../acle.rst#neon-sve-bridge>`_. These C intrinsics enable the
```

Please double check it works by browsing the rendering at https://github.com/MDevereau/acle/blob/next-release/main/design_documents/ARM_NEON_SVE_BRIDGE.rst#design-document-for-arm_neon_sve_bridge